docs/swagger.yaml

```diff
@@ -1075,6 +1151,97 @@ paths:
               $ref: '#/definitions/DetailedTag'
         '500':
           description: Unexpected internal errors.
+  '/repositories/{repo_name}/tags/{tag}/labels':
+    get:
+      summary: Get labels of an image.
+      description: |
+        Get labels of an image specified by the repo_name and tag.
+      parameters:
+        - name: repo_name
+          in: path
+          type: string
+          required: true
+          description: The name of repository.
+        - name: tag
+          in: path
+          type: string
+          required: true
+          description: The tag of the image.
+      tags:
+        - Products
+      responses:
+        '200':
+          description: Successfully.
+        '401':
+          description: Unauthorized.
+        '403':
+          description: Forbidden.
+        '404':
+          description: Resource not found.
+    post:
+      summary: Add a label to image.
+      description: |
+        Add a label to the image.
+      parameters:
+        - name: repo_name
+          in: path
+          type: string
+          required: true
+          description: The name of repository.
+        - name: tag
+          in: path
+          type: string
+          required: true
+          description: The tag of the image.
+        - name: label
+          in: body
+          description: Only the ID property is required.
+          required: true
+          schema:
+            $ref: '#/definitions/Label'
+      tags:
+        - Products
+      responses:
+        '200':
+          description: Successfully.
+        '401':
+          description: Unauthorized.
+        '403':
+          description: Forbidden.
+        '404':
+          description: Resource not found.
+  '/repositories/{repo_name}/tags/{tag}/labels/{label_id}':
+    delete:
+      summary: Delete label from the image.
+      description: |
+        Delete the label from the image specified by the repo_name and tag.
+      parameters:
+        - name: repo_name
+          in: path
+          type: string
+          required: true
+          description: The name of repository.
+        - name: tag
+          in: path
+          type: string
+          required: true
+          description: The tag of the image.
+        - name: label_id
```

Comment:
This is the `resource_label_id` right? not the `label_id`.

Reply:
It's `label_id`.

Reply:
I don't fully agree but given this is how github works we can leave this implement as-is.